### PR TITLE
Add isLightDescendant extern.

### PR DESCRIPTION
**Pull request by tommycli**
@jklein24 Sorry to bother. But I need to fix a typo and add another extern to get chromium to closure compile. Moving target and so forth.

### CODE REVIEW

File: polymer.externs.js [hunk 1]
```diff
@@ -597,11 +604,11 @@ PolymerDomApi.prototype.removeAttribute = function(attribute) {};
 
 /**
  * @param {!Function} callback
- * @return {!{fn: !Function=, _nodes: !Array<!Node>}}
+ * @return {!{fn: !Function, _nodes: !Array<!Node>}}
```
Comment (jklein24) [inline polymer.externs.js]: Ah sorry this was my mistake in the comment on your previous CL. This should really be

`{!{fn: (!Function|undefined), _nodes: !Array<!Node>}}`

same below. Confusing, I know :-).

File: polymer.externs.js [hunk 2]
```diff
@@ -438,6 +438,13 @@ PolymerElement.prototype.translate3d = function(x, y, z, node) {};
 PolymerElement.prototype.importHref = function(href, onload, onerror) {};
 
 /**
+ * Checks whether an element is in this element's light DOM tree.
+ * @param {HTMLElement=} node The element to be checked.
+ * @return {Boolean} true if node is in this element's light DOM tree.
```
Comment (jklein24) [inline polymer.externs.js]: s/Boolean/boolean/

File: polymer.externs.js (hunk 1 above)
Comment (jklein24) [inline polymer.externs.js]: Resolved this in https://github.com/PolymerLabs/polymer-externs/pull/37

**Comment by tommycli** [comment]
Fixed, thanks!

**Comment by tommycli** [comment]
@jklein24 Thanks! Can you also roll a new release?

**Pull request merged**